scrapers/CorbinFisher.yml

```diff
@@ -0,0 +1,28 @@
+name: corbinfisher
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - corbinfisher.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1
+      Date:
+        selector: //div[@class="added"]
+        postProcess:
+          - replace:
+              - regex: (^\S+\s+)(.+\d{4})(.+)
+                with: $2
+          - parseDate: January 2, 2006
+      Performers:
+        Name: //div[@class="modelFeaturing"]/a | //div[@class="modelFeaturing"]//a
```

Comment:
`//div[@class="modelFeaturing"]//a` includes the `//div[@class="modelFeaturing"]/a` result.  Was there a reason for the union and not just using the second (//a) selector?
I would also add the URL field as for most performers only the name part is used
```yaml
        Name: //div[@class="modelFeaturing"]//a
        URL: //div[@class="modelFeaturing"]//a/@href
```